### PR TITLE
Add Day 1 schedule for Berlin 2019

### PR DESCRIPTION
**Pull request by keywordnew**

### CODE REVIEW

File: 2019-05-Berlin/agenda.md [hunk 1]
```diff
@@ -32,15 +30,30 @@ Axel-Springer-Str. 55
...
+| 1200 | | 🍍 Lunch | | | | 
+| 1230 | Main | _previous session continues_ | | | | 
+| 1300 | Main | OpenJS Foundation CPC Session | Jory, Myles, &/or Tracy Hinds, or new CPC Director or Chairperson | | | 
+| 1330 | Main | _previous session continues_ | | | | 
+| 1400 | Main | Security, modules and Node.js | Guy Bedford | | | 
+| 1430 | Main | | | | | 
+| 1500 | Main | Fetch in Node.js | @MylesBorins @mcollina | | | 
+| 1530 | Main | ☕️ coffee break | | | | 
+| 1600 | Main | _previous session continues_ | | | | 
+| 1630 | Main | Intro, Sponsor Talk, Code of Conduct | organizers | | |
```
Comment (mcollina) [inline 2019-05-Berlin/agenda.md]: I don't think this is correct, this should be the first session of the day.

**Comment by WaleedAshraf** [comment]
@here can someone give me push access? I have added more session from the sheet.

**Comment by WaleedAshraf** [comment]
@keywordnew @jorydotcom added a session for day 1.
Few notes,

1: Website redesign got moved to day 2, needs to be updated here.
2: from 12:00 to 12:30 on Day 1, which session is going on? There's nothing on sheet. Also, rows are collapsed. I think they shouldn't be collapsed? Because next session CFP starts at 13:00

**Comment by WaleedAshraf** [comment]
merging for now, we can update later.

**Pull request merged**